Tutorial/Dialogue/co_011014.csv

```diff
@@ -0,0 +1,3 @@
+name01#0,"""Telepipes"""
```

Comment:
co_011014.csv's lines `linewindow#0` and `text00#0` are missing

Reply:
also `name01#0` does not exist in this CSV

Reply:
think fixed